### PR TITLE
[WIP] Fuzzers stabilization

### PR DESCRIPTION
**Pull request by tmichalak**
This PR is solely for the purpose of running CI builds in order to experiment with some changes that should improve the stability of the fuzzer runs (#661)

### CODE REVIEW

**Comment by litghost** [comment]
Please fix the commit DCO?

**Comment by tmichalak** [comment]
@mithro The CI runs seem stable lately, but the database for kintex in prjxray-db has to be updated as it's still failing on the diffs between the new and previous master db values.

File: fuzzers/056-pip-rem/generate.tcl [hunk 1]
```diff
@@ -1,5 +1,9 @@
 source "$::env(XRAY_DIR)/utils/utils.tcl"
 
+proc waive_drc_errors {} {
```
Comment (mithro) [inline fuzzers/056-pip-rem/generate.tcl]: This needs a comment about why it is enabled.

Comment (mithro) [inline fuzzers/056-pip-rem/generate.tcl]: `waive_drc_errors` would be better called `demote` or `disable`...

**Comment by mithro** [comment]
This pull request needs to be split into multiple pull requests and the parts merged ASAP.

File: fuzzers/056-pip-rem/generate.tcl (hunk 1 above)
Comment (tmichalak) [reply fuzzers/056-pip-rem/generate.tcl]: Created a separate PR #787 and applied your comment there.

Comment (tmichalak) [reply fuzzers/056-pip-rem/generate.tcl]: Created a separate PR #787 and applied your comment there.

**Comment by tmichalak** [comment]
Closing this experimental PR. The problematic fuzzers are being disabled in PR #788.
New PR will be opened for re-enabling the specific fuzzers.

**Pull request closed without merging**